sale_sourced_by_line/__openerp__.py

```diff
@@ -1,27 +1,26 @@
 # -*- coding: utf-8 -*-
 # Copyright 2013-2014 Camptocamp SA - Guewen Baconnier
-# © 2015 Eficent Business and IT Consulting Services S.L.
-# - Jordi Ballester Alomar
-# © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
-# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
     'name': 'Sale Sourced by Line',
     'summary': 'Multiple warehouse source locations for Sale order',
     'version': '9.0.1.0.0',
     "author": "Camptocamp,"
               "Eficent,"
-              "Serpent.,"
+              "Serpent,"
               "Odoo Community Association (OCA)",
```

Comment:
@JayVora-SerpentCS Yes we tried, but that violates the constraint of 128 characters for the author. So I cannot pass the full names here.